packages/framework-core/src/booster-app.ts

```diff
@@ -9,27 +9,26 @@ import {
 } from '@boostercloud/framework-types'
 import { Importer } from './importer'
 import { buildLogger } from './booster-logger'
-import { BoosterEventDispatcher } from './booster-event-dispatcher'
-import { BoosterAuth } from './booster-auth'
 import { fetchEntitySnapshot } from './entity-snapshot-fetcher'
-import { BoosterGraphQLDispatcher } from './booster-graphql-dispatcher'
-import { BoosterSubscribersNotifier } from './booster-subscribers-notifier'
-import { BoosterScheduledCommandDispatcher } from './booster-scheduled-command-dispatcher'
 
 /**
- * Main class to interact with Booster and configure it.
- * Sensible defaults are used whenever possible:
- * - `provider`: `Provider.AWS`
- * - `appName`: `new-booster-app`
- * - `region`: 'eu-west-1'
- *
+ * Static (singleton) class that provides access to the logger, the config,
+ * and the runtime methods to configure, manage and interact with Booster features.
  */
-export class Booster {
+export class BoosterApp {
+  /**
+   * Set of environment names set from the application using `Booster.configure('environment_name', ...)`
+   */
   public static readonly configuredEnvironments: Set<string> = new Set<string>()
-  private static logger: Logger
+
+  /**
+   * Default logger, as configured in the current
```

Comment:
I think there is something missing after `... in the current`

Reply:
I have no idea, so...
```suggestion
   * Default logger
```